tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot

```diff
@@ -45,13 +45,27 @@ No-trunc images
     Length Should Be  @{line}[2]  64
 
 Specific images
-    ${status}=  Get State Of Github Issue  2248
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-3-Docker-Images.robot needs to be updated now that Issue #2248 has been resolved
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images alpine:3.1
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Not Contain  ${output}  Error
-#    Should Contain  ${output}  3.1
-#    Should Contain X Times  ${output}  alpine  1
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images alpine:3.1
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    Should Contain  ${output}  3.1
+
+Filter images before
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images -f before=alpine
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    @{lines}=  Split To Lines  ${output}
+    Length Should Be  ${lines}  3
+    Should Contain  ${output}  3.1
+
+Filter images since
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images -f since=alpine:3.1
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    @{lines}=  Split To Lines  ${output}
+    Length Should Be  ${lines}  3
+    Should Contain  ${output}  latest
+
 
```

Comment:
I know it's tough, but you weren't able to find an image that has a label?  I think I tried to find one in the past and couldn't find one.  I assume you left out label filtering test because of this?